crates/ruma-client-api/src/dehydrated_device.rs

```diff
@@ -0,0 +1,88 @@
+//! Endpoints for managing devices.
```

Comment:
```suggestion
//! Endpoints for managing dehydrated devices.
```
Or something like that